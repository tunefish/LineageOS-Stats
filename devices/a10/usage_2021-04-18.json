{
  "10.3": 1,
  "16.0": 69,
  "17.1": 915,
  "18.1": 520,
  "none": 449
}