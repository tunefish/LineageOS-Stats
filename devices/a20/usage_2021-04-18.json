{
  "17.1": 2434,
  "18.1": 237,
  "none": 28
}