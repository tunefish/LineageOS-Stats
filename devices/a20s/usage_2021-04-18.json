{
  "15.1": 1,
  "17.1": 2494,
  "none": 1
}