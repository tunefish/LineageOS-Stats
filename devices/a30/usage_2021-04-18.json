{
  "17.1": 37,
  "18.0": 758,
  "18.1": 441,
  "none": 4
}