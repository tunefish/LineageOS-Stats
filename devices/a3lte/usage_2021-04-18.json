{
  "15.1": 743,
  "16.0": 275,
  "17.1": 933
}