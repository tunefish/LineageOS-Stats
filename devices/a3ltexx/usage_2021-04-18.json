{
  "14.1": 1173
}