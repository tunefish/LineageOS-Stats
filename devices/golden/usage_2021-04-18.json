{
  "13.0": 1318,
  "14.1": 4590,
  "15.1": 6,
  "17.1": 9,
  "none": 1
}